package.json

```diff
@@ -18,6 +18,9 @@
     "test": "jest src",
     "test-e2e": "jest test-e2e",
     "test-sync": "bash test-e2e/sync/launch.sh",
+    "flow-typed": "npm run install-flow-typed && npm run clean-flow-typed",
+    "install-flow-typed": "flow-typed install -s --overwrite",
+    "clean-flow-typed": "rm -f flow-typed/npm/{reselect_*,redux-actions_*,react-redux_*,react-i18next_v7.x.x.js,styled-components_v3.x.x.js,winston*}",
```

Comment:
can you rename to something like `remove breaking-flow-typed` ? to make it more obvious it's not about removing the flow-typed before reinstalling

Reply:
ok